index.js

```diff
@@ -0,0 +1,214 @@
+import * as Sentry from '@sentry/node'
+import getRawBody from 'raw-body'
+import httpAssert from 'http-assert'
+import * as ethers from 'ethers'
+import { json, status } from 'http-responders'
+import assert from 'node:assert'
+
+const maxScore = BigInt(1e15)
+// https://github.com/filecoin-station/spark-impact-evaluator/blob/fd64313a96957fcb3d5fda0d334245601676bb73/test/Spark.t.sol#L11C39-L11C65
+const roundReward = 456621004566210048n
```

Comment:
I guess it's okay to hard-code this value for a short-living service. A more robust solution would be to read the round reward from the smart contract whenever we are mapping scores to rewards.

Reply:
Agreed